src/components/TextBlockWithLabel/TextBlockWithLabel.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC, ReactElement, isValidElement } from "react";
+import styled from "styled-components";
+import { P8, P5 } from "@deskpro/app-sdk";
+
+export type Props = {
+  label?: string | ReactElement;
+  text?: string | number | ReactElement;
+  marginBottom?: number;
+};
+
+const Container = styled.div<Props>`
+  margin-bottom: ${({ marginBottom }) => `${marginBottom}px`};
+`;
+
+const Label = styled(P8)`
+  color: ${({ theme }) => theme.colors.grey80};
+`;
+
+const TextBlockWithLabel: FC<Props> = ({ text, label, marginBottom = 10 }) => {
```

Comment:
There is a similar component in the **app-sdk**. Use this `import { Property } from "@deskpro/app-sdk";`